components/breadcrumbs/breadcrumb.js

```diff
@@ -0,0 +1,89 @@
+import '../icons/icon.js';
+import '../link/link.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class BreadCrumb extends RtlMixin(LitElement) {
```

Comment:
For consistency I think the the `C` should be lowercase: `class Breadcrumb ...`

Reply:
Done ✔️

Reply:
Done ✔️